modules/api/src/main/java/com/intuit/wasabi/api/AssignmentsResource.java

```diff
@@ -152,18 +155,18 @@ public Response getAssignment(
         Assignment assignment = getAssignment(userID, applicationName, experimentLabel, context, createAssignment,
                 ignoreSamplingPercent, null, headers);
 
-        return httpHeader.headers().entity(toMap(assignment)).build();
+        return httpHeader.headers().entity(toSingleAssignmentResponseMap(assignment)).build();
     }
 
     private Assignment getAssignment(final User.ID userID, final Application.Name applicationName,
```

Comment:
Cosmetic: Suggest changing the method name to getSingleAssignment as it is just wrapping a specific assignment type.